ipfx/stimulus.py

```diff
@@ -26,7 +26,7 @@ def has_tag(self, tag, tag_type=None):
 
 class StimulusOntology(object):
 
-    DEFAULT_STIMULUS_ONTOLOGY_FILE = os.path.join(os.path.dirname(__file__), 'defaults/stimulus_ontology.json')
+    DEFAULT_STIMULUS_ONTOLOGY_FILE = os.path.join(os.path.dirname(__file__), './defaults/stimulus_ontology.json')
```

Comment:
probably don't want to hardcode these forward slashes

Reply:
what about the one between defaults and stimulus ontology?

Reply:
I am not sure how to point to the json otherwise